README.md

```diff
@@ -1,2 +1,33 @@
 # hw01
+
 Repository for homework 01
+
+## Claire Brainard Bio
+
+Here is a list of things I enjoy doing:
+
+  * Baking Bread
+  
+  * Hiking
+  
+  * Camping
+  
+  * Traveling, esp. roadtrips!
+  
+  * Making Kombucha
+  
+  
+  **After** I finish up my Maters degree here at UChicago, I am moving back to *Southern California*!
+  
+### Below is a photo of my cat, Marlon:
+![](6E521B46-F262-4C2E-96A2-6DEB84E6B3F6.jpg)
```

Comment:
If you want, you can also put the words you'd like to appear in case the picture source is not working, like alternative sources, or "picture missing" in the []